firebase.js

```diff
@@ -0,0 +1,21 @@
+// Import the functions you need from the SDKs you need
+import { initializeApp } from "firebase/app";
+import { getFirestore } from "firebase/firestore";
+// TODO: Add SDKs for Firebase products that you want to use
+// https://firebase.google.com/docs/web/setup#available-libraries
+import { getAuth } from "firebase/auth";
+
+// Your web app's Firebase configuration
+const firebaseConfig = {
+  apiKey: "AIzaSyAXjYmRpzu5-Wq3V774aIuMBjzy_Hn2Axw",
```

Comment:
Those config are exposed this way in the code base. A better way is to store them as environmental variables.

Reply:
Thanks for the comment. Yes makes total sense, will store them as environmental variables.